templates/blog/index.html

```diff
@@ -1,5 +1,98 @@
 {% extends "_layout.html" %}
 
+{% block meta_title %}Blog | Snapcraft.io{% endblock %}
```

Comment:
I don't think we ever call site 'Snapcraft.io', title should just say 'Blog | Snapcraft' I guess.